libs/openFrameworks/3d/ofEasyCam.h

```diff
@@ -219,22 +222,21 @@ class ofEasyCam : public ofCamera {
 
     /// \brief The current rotation quaternion.
 	glm::quat curRot;
-
-    /// \brief The previous X axis.
-	glm::vec3 prevAxisX;
-
-    /// \brief The previous Y axis.
-	glm::vec3 prevAxisY;
-
-    /// \brief The previous Z axis.
-	glm::vec3 prevAxisZ;
-
-    /// \brief the previous camera position.
-	glm::vec3 prevPosition;
-
-    /// \brief The previous camera orientation.
-	glm::quat prevOrientation;
-
+	
+	/// \name On Press cache
+	/// \{
+	/// \brief camera properties when the mouse is pressed.
+
+	glm::vec3 onPressAxisX;
```

Comment:
right. 
actually there were some of these vars called lastSomething that were used to hold the values of Something when the mouse gets pressed. In particular there were prevMouse, which was storing the mouse position of the previous frame, and lastMouse which was storing the position the mouse had when it got pressed. this naming pattern seemed really confusing, so I decided to change the lastMouse for onPressMouse. 
Although I see your point. Maybe `lastPressMouse` would be better?